.travis.yml

```diff
@@ -2,21 +2,16 @@ language: node_js
 sudo: false
 dist: precise
 node_js:
+    - "node"
     - "7"
     - "6"
     - "5"
     - "4"
     - "0.12"
-before_install:
-- npm install npm@2 -g
-- npm install npm -g
```

Comment:
Not sure why these had to be added but the commits are from 2years ago so if everything works on travis... its probably fine?